### ISSUE TITLE
Where is unit tests of this package?

### ISSUE THREAD

**Issue by abdennour** [razagill/react-phone-input#35]
We cannot consider a software is mature until unit-tests are written and covered at least 80% of code. 

Let's write it and get badges from coverall.io accordingly

**Comment by sample-usr** [comment]
@abdennour yes that's something I was meaning to get into. Please share any rough ideas you have to start with.

**Comment by abdennour** [comment]
Please take a look at unit-tests of [react-csv](
https://github.com/abdennour/react-csv) component ,It might be useful for this component:

**Comment by kimeshan** [comment]
@razagill You can unit test react components using [Jest](https://facebook.github.io/jest/) and [Enzyme](https://github.com/airbnb/enzyme)
